server/app.js

```diff
@@ -0,0 +1,6 @@
+require('babel-register')({
+  presets: ['env'],
+});
+
+// Import the rest of our application.
+module.exports = require('./server.js');
```

Comment:
Part of the requirement of the project is that you use ES6 syntax, I see some ES5 in your codebase.